### ISSUE TITLE
Add a `--create-empty-files` option

### ISSUE THREAD

**Issue by gromenauer** [manga-py/manga-py#386]
# Feature request

Add a `--create-empty-files` option, with this option all new files will have a 0 bytes size

**Is your feature request related to a problem? Please describe.**

Suppose:
* You follow a manga in provider PROVIDER1 until chapter 10 
* You want to switch to provider PROVIDER2 that have a distinct name convention (say PROVIDER1 disappeared) 
* You don't want chapter 1 to 10 from PROVIDER2

Solution:  
* Create chapter 1 to 10 with new name convention as empty files. It will get chapters from 11 onwards when they appear.

**Describe the solution you'd like**

Add a `--create-empty-files` option, with this option all new files will be created with 0 bytes size. (If you create too many, they are easy to delete).

**Describe alternatives you've considered**
🤔 ... None 😅

**Additional context**
🦗 🦗

**Comment by 1271** [comment]
Do you mean chapters to skip?

**Comment by gromenauer** [comment]
Yes. It's a easy way to avoid to remember to use `--skip-volumes` for every manga that changed provider. And zero byte files uses almost no space.
Some times even in the same provider the naming style changes. Then the zero byte files is a no brainer to avoid duplicates, renaming the old style files is too much hassle.
